src/apppstatus.cc

```diff
@@ -35,11 +35,14 @@ extern ref<YPixmap> taskbackPixmap;
 
 class NetDevice : public INetDevice {
 public:
-    NetDevice(mstring netdev) : fDevName(netdev) {}
+    NetDevice(mstring netdev) : fDevName(netdev) {
+        should_display_flag = netstatusShowOnlyRunning ? IFF_RUNNING : IFF_UP;
```

Comment:
This can be done in the member initializer, but @gijsbers might have a different opinion.